src/Data/ProgrammesDb/EntityRepository/CoreEntityRepository.php

```diff
@@ -636,6 +631,36 @@ public function coreEntityAncestryGetter(array $ids): array
         return $cached;
     }
 
+    private function findByPidFullCommon(string $pid, string $entityType = 'CoreEntity'): QueryBuilder
```

Comment:
Should `$entityType` have a default? Why not just make it required?

Reply:
Yeah, fair point.